hyperspy/tests/io/test_hdf5.py

```diff
@@ -707,3 +707,20 @@ def test_save_chunks_signal_metadata():
     with tempfile.TemporaryDirectory() as tmp:
         filename = os.path.join(tmp, 'test_save_chunks_signal_metadata.hspy')
     s.save(filename, chunks=(5, 5, 10))
+
+
+def test_chunking_saving_lazy():
```

Comment:
This test should also check for the case of `s.save("filename.hspy", chunks=(3, 3, 10, 10))`, ergo where the chunk size is specified.

Reply:
This was already covered in the non lazy case, but it doesn't hurt to add a test for this in the lazy case too!